/**:
  ros__parameters:
    update_hz: 10.0 # [Hz]
    initial_covariance: 0.5
    initial_offset: 0.0 # [rad]
    process_noise_covariance: 1.0e-6
    measurement_noise_covariance: 0.5
    denominator_floor: 1.0e-12
    covariance_floor: 1.0e-12
    min_velocity: 1.0 # [m/s]
    max_steer: 0.03 # [rad]
    max_steer_rate: 0.02 # [rad/s]
    max_ang_velocity: 0.02 # [rad/s]
    max_steer_buffer: 2.0 # [s]
    max_pose_lag: 0.5 # [s]

    calibration:
      mode: "off"  # Options: ["off", "manual", "auto"]

      # Logic hresholds
      error_threshold: 0.005        # min error to trigger an AUTO update [rad]
      covariance_threshold: 0.00075 # max uncertainty to trust the estimate

      # Persistence & Safety
      min_steady_duration: 10.0     # required duration of continuous estimation data [s]
      min_velocity: 1.0             # minimum speed to allow estimation updates [m/s]
      max_offset_limit: 0.05        # safety cap to prevent extreme calibration jumps [rad]
      min_update_interval: 100.0    # minimum duration between updates [s]
